core-java-modules/core-java-lang-6/pom.xml

```diff
@@ -2,15 +2,17 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="http://maven.apache.org/POM/4.0.0"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Hi @Bipinkumar27 

Let's remove the line spacing for modelVersion, artifactId, name, version, packaging.

We only want line spacing for parent, dependencies, build and properties